### ISSUE TITLE
cull_idle_server won't start (error from jupyterhub.log included)

### ISSUE THREAD

**Issue by huntrods** [jupyterhub/jupyterhub#2373]
I installed cull_idle_servers as per the instructions. I copied cull_idle_servers.py into the directory that also contains my jupyterhub_config.py script (/srv/jupyterhub) and then added the appropriate lines to jupyterhub_config.py as per the instructions.

I've tried several times now to start Jupyterhub (I use supervisorctl) and the logs always show the same error  (attached):
root@rsh_ubuntu_server:/srv/jupyterhub# supervisorctl restart jupyterhub
jupyterhub: stopped
jupyterhub: started
root@rsh_ubuntu_server:/srv/jupyterhub# tail /var/log/jup*.log
05:06:57.014 - info: [ConfigProxy] 200 GET /api/routes
[I 2019-01-08 05:06:57.015 JupyterHub proxy:301] Checking routes
[I 2019-01-08 05:06:57.015 JupyterHub proxy:370] Adding default route for Hub: / => http://127.0.0.1:8081
05:06:57.019 - info: [ConfigProxy] Adding route / -> http://127.0.0.1:8081
[I 2019-01-08 05:06:57.021 JupyterHub app:1912] JupyterHub is now running at http://:4434/
05:06:57.022 - info: [ConfigProxy] 201 POST /api/routes/
  File "cull_idle_servers.py", line 351
    pass
       ^
TabError: inconsistent use of tabs and spaces in indentation
root@rsh_ubuntu_server:/srv/jupyterhub#

The 'pass' is the last line of the cull_idle_servers.py script. It seems properly indented, and it's the last line of the file. I even tried adding a blank line after 'pass', but it always fails with the above error. What shall I do to fix this? Thanks.

**Comment by huntrods** [comment]
Oh, one other quick comment, not mentioned in the documentation: you need to "import sys" in your jupyterhub_config.py if it hasn't been used before (as in my case) or JupyterHub won't even start.

**Comment by huntrods** [comment]
FIXED IT!
The original file I used 'cull_idle_servers.py' was created using cut-and-paste from the notebook file here. For some reason the line with 'pass' didn't have a tab. That generated the tab error in the JupyterHub log the first time. What is bizarre is I saw that and edited the file on the server, but the error remained. That is when I posted here.
Today I looked again at the file, and in a slightly better editor saw that the line 'pass' needs TWO tabs to indent properly. I know on the server I only added one tab, but it appeared to line up so I thought it was OK.

Today I copied the local file with proper (two) tabs on that last line to my server, restarted JupyterHub and it's running with no errors.
YAY!

Apologies for "operator error".
